dashboard/app/views/home/_homepage.html.haml

```diff
@@ -27,19 +27,21 @@
 #homepage-container
 
 #admin-extras
-  -if current_user.admin?
-    %a{href: CDO.code_org_url('/ops-dashboard#/districts'), style: 'text-decoration: none;'}
-      %h2
-        Manage Districts
-  -if current_user.admin? || has_permission?('facilitator')
-    %a{href: CDO.code_org_url('/ops-dashboard#/workshops'), style: 'text-decoration: none;'}
-      %h2
+  -if current_user.admin? || current_user.permission?(UserPermission::CREATE_PROFESSIONAL_DEVELOPMENT_WORKSHOP) || current_user.permission?(UserPermission::WORKSHOP_ADMIN) || current_user.permission?(UserPermission::DISTRICT_CONTACT)
```

Comment:
Why district contact? I don't think that should be in this list.

What about our `CanCan` ability check? That was cleaner. Did something not work?

If we are doing it this way, this line is getting long and cumbersome. This might better belong in a helper.